dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+pytest
```

Comment:
Ah, that's pretty cool. That looks like it only pins production requirements at release time though. I find it equally as important to pin dev requirements because this is the environment that breaks more often for other developers.